### DISCUSSION TITLE
Widgets expand to the same size as neighbor

### DISCUSSION THREAD

**Discussion by Nialisc** [elkowar/eww#718] [Q&A] [answered]
Hey there !
So, I'm trying to create a widget with two `boxes` as children, one for a label, and the other for a value.

The issue here is that each `box` widget expand to the same size as the other widget on the same nested level. (cf. screenshot)
But I want them to the size of their "content" to not have labels (icons) with such width for nothing.
Here is my code https://github.com/Nialisc/shangrila/tree/main/modules/apps/gui/eww/config

Do you have any idea of what's going on here ?

![bar](https://user-images.githubusercontent.com/87083963/227247641-b3e5143d-1f5c-4385-9d68-4ff596888f22.png)

**Comment by viandoxdev** [comment, marked as answer]
This behavior is caused by space-evenly (which defaults to true) on the parent box, disabling it `:space-evenly false` should solve this

**Reply by Nialisc** [reply]
Works like a charm, thank you :pray:
